crashtracker/src/configuration.rs

```diff
@@ -31,27 +55,21 @@ impl CrashtrackerConfiguration {
         collect_stacktrace: bool,
         create_alt_stack: bool,
         endpoint: Option<Endpoint>,
-        path_to_receiver_binary: String,
         resolve_frames: CrashtrackerResolveFrames,
-        stderr_filename: Option<String>,
-        stdout_filename: Option<String>,
         timeout: Duration,
     ) -> anyhow::Result<Self> {
-        anyhow::ensure!(
-            !path_to_receiver_binary.is_empty(),
-            "Expected a receiver binary"
-        );
-        anyhow::ensure!(stderr_filename.is_none() && stdout_filename.is_none() || stderr_filename != stdout_filename,
-        "Can't give the same filename for stderr and stdout, they will conflict with each other"
-    );
+        //     anyhow::ensure!(
+        //         !path_to_receiver_binary.is_empty(),
+        //         "Expected a receiver binary"
+        //     );
+        //     anyhow::ensure!(stderr_filename.is_none() && stdout_filename.is_none() ||
+        // stderr_filename != stdout_filename,     "Can't give the same filename for stderr
```

Comment:
Whimsical comment follows.

In many cases, stderr and stdout are already interleaved by the time they get to the user (either their terminal, or an aggregation platform that just rips everything _similarly_ to a terminal).  This system uses `dup2()` in order to duplicate the file descriptors, so the interleaving would actually happen at the level of the `write()` syscall.  In userland, even libc doesn't know better and will provide two separate buffers to the singular file descriptor, but that doesn't decrease the safety of the operation.

I sense that this situation may actually be fine (in the sense that it is precisely what the user asked for and should have no _unwanted_ side-effects).  Of course, if someone complains that this interleaves the outputs in a confusing way, then they deserve the stick-bicycle meme.  Possibly twice.

Reply:
As noted below, we're creating them with `File::create`, so the same name twice would be an issue.  If we want to allow interleaving, I can change the code to allow that in a subsequent PR